docs/modules/ROOT/pages/mcp.adoc

```diff
@@ -0,0 +1,34 @@
+= Model Context Protocol
```

Comment:
I think this doc should also have a more complete example with an AiService that uses it

Reply:
The tricky bit is that the AiService does not have to do anything at all specific to MCP, as the last paragraph says, you can just use the configuration approach and the tools will be available to all AI services that don't explicitly  declare a different tool provider

Reply:
Agreed, but it's buried in the last part of the doc while it's arguably the most important part, so I think we need to make it more prominent

Reply:
I've added a sentence to the start of the section